course/src/main/java/org/openedx/course/presentation/detail/CourseDetailsFragment.kt

```diff
@@ -544,6 +550,7 @@ private fun CourseDescription(
     onWebPageLoaded: () -> Unit
 ) {
     val context = LocalContext.current
+    val config = getKoin().get<Config>()
```

Comment:
why config is injected here, it need to be injected inside viewModel, and all config fields should be provided throw uiState into UI